woocommerce/woocommerce-ios
# Refunded Products: Display refunded product cells in refunded products screen

Part 1 of #1703. 

This PR displays the section title and order item cells for products that have been refunded. 

## Notes 
1. This PR does not update the order item quantities in the Order Details page. Coming soon!
2. This PR does not combine order items into a single cell. Coming soon!
3. This PR does not allow pull-to-refresh on the Refunded Products view controller. Coming soon!

### Screenshots
**Design**
<img width="436" alt="Screen Shot 2020-01-07 at 11 43 13 AM" src="https://user-images.githubusercontent.com/1062444/71916573-dab3ae80-3143-11ea-8db3-385bf9866750.png">

**Implementation**
iPhone Xs
<img src="https://user-images.githubusercontent.com/1062444/71916132-df2b9780-3142-11ea-94b3-b910fef6f41a.png" width="350" />

### To Test

1. On the website, choose an order and create a refund by item. Refund 2 items or more and add a note. (Make it a partial refund by refunding only a few items in the order. Don't change the order status to Refunded.) After creating the refund, go to the app and navigate to the Order Details for the chosen order.
2. On Order Details, the "Refunded Products" section title should appear and a "2 Items" detail cell should be in this section. 
3. Tap the "2 Items" cell and navigate to the Refunded Products detail screen. The title should appear. The back button should not have any words. The section titles and cells should appear.

Update release notes:
